.husky/commit-msg

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+if [[ -n $HUSKY_BYPASS ]]; then
+  echo "Husky bypassed"
+  exit 0;
+fi
+
+yarn commitlint --config ./config/commitlint.config.js --edit $1
```

Comment:
What is this for?

Reply:
It's for ensuring commit message will respect conventionalcommits